src/vs/base/parts/tree/browser/treeModel.ts

```diff
@@ -974,6 +976,7 @@ export class TreeModel {
 	public getInput(): any {
 		return this.input ? this.input.getElement() : null;
 	}
+
```

Comment:
blank line can be removed

Reply:
The styling of the remaining code file has lines there so keeping it for now.

Reply:
We shouldn't introduce unnecessary changes to VS Code source code because it makes the merges more difficult